.eslintrc.cjs

```diff
@@ -52,12 +36,16 @@ module.exports = {
     es2017: true,
     node: true,
   },
+  globals: {
+    App: 'readonly',
```

Comment:
Is this just "the fix" now? Or do we want to leave that "Temporary fix, see the following:" note in there in case there is an actually fix at some point?

Also, just want to confirm we don't want `NetworkError` and `NextPageToken` there anymore?

Reply:
Yeah, we don't need those anymore cause they are our types and we aren't referencing them globally anymore, i.e. we're importing them wherever they're needed.

And good call, I'll add a comment to the issue I linked in other comments in this PR